### DISCUSSION TITLE
Pushover poster support

### DISCUSSION THREAD

**Discussion by miguelsousa46** [sct/overseerr#3741] [Q&A] [answered]
Hi guys, I've configured pushover notifications in overseerr, working fine but with no poster, just plain text, is it supported? If so is there any extra configuration? Apart from the pushover tokens I m not seeing anything else.

**Comment by nmbrg** [comment]
I’m getting posters on my notifications. Don’t think I’ve made any special settings. Though I did not get posters before switching from docker to a proxmox LXC install. Might be that the proxmox LXC install is running the dev build instead of beta. Or perhaps pushover app has been updated with new functionality, either way I’m now getting posters that I did not get before.

**Reply by miguelsousa46** [reply]
I´m on a Docker install, going to try different images if it changes some how.

**Comment by miguelsousa46** [comment, marked as answer]
Installed de develop image and now the poster image is working!
